dgbpy/torch_modelinfo.py

```diff
@@ -31,13 +39,15 @@ def __num_outputs( torch_model ):
     return len(__output_names(torch_model))
 
 def model_info( modelfnm ):
-    model = torch.load( modelfnm )
+    model = torch.jit.load( modelfnm )
```

Comment:
here, disable warnings that may be printed if modelfnm is not a torchscript model